operations/src/main/java/com/nextdoor/bender/operation/json/array/ArraySplitOperation.java

```diff
@@ -60,9 +69,12 @@ public ArraySplitOperation(String path) {
       ArrayList<InternalEvent> output = new ArrayList<InternalEvent>();
       for (JsonElement elm : arr) {
         try {
-          InternalEvent newEvent =
-              new InternalEvent(elm.toString(), ievent.getCtx(), ievent.getArrivalTime());
-          DeserializedEvent newDeserEvent = new GenericJsonEvent(elm.getAsJsonObject());
+          JsonObject newObject = elm.deepCopy().getAsJsonObject();
```

Comment:
I don't think a deep copy is required here. We can just modify `elm` because at this point we won't be using `elm` anywhere else.

Reply:
right, changed to elm.getAsJsonObject()